jmclient/jmclient/configure.py

```diff
@@ -323,6 +323,14 @@ def jm_single():
 # this is the minimum satoshis per vbyte we allow in the payjoin
 # transaction; note it is decimal, not integer.
 min_fee_rate = 1.1
+
+# for payjoins to hidden service endpoints, the socks5 configuration:
+socks5_host = localhost
+socks5_port = 9050
```

Comment:
Maybe better naming for these would be `onion_socks5_host` and `onion_socks5_port`, as it is just to connect to .onion URL's, not generic SOCKS5 proxy for all outgoing connections?

Reply:
I mean this is just in the PAYJOIN section so doesn't seem necessary?

Reply:
We could add support for other networks in the future, for example, one way to connect to `.i2p` URL's is also via SOCKS5 proxy.

Reply:
OK

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/638/commits/1983e0886c3cfffee71ef2bfd6fd26c1924c54d4